docs/content/concepts/repositories-workspaces/workspaces.mdx

```diff
@@ -1,201 +1,281 @@
 
-A _workspace_ is a collection of user-defined [repositories](/concepts/repositories-workspaces/repositories) and information about where to find them. Dagster tools, like Dagit and the Dagster CLI, use workspaces to load user code.
+A workspace is a collection of user-defined [code locations](/concepts/code-locations) and information about where to find them. Workspaces can accept code locations defined using a <PyObject object="Definitions" /> object or a <PyObject object="repository" decorator /> decorator.
 
-## Relevant APIs
-
-| Name                                       | Description                                                                                                   |
-| ------------------------------------------ | ------------------------------------------------------------------------------------------------------------- |
-| <PyObject object="repository" decorator /> | The decorator used to define repositories. The decorator returns a <PyObject object="RepositoryDefinition" /> |
-
-## Overview
-
-A Dagster Workspace can contain multiple [repositories](/concepts/repositories-workspaces/repositories) sourced from a variety of different locations, such as Python modules or Python virtualenvs.
+Prior to the introduction of <PyObject object="Definitions" />, workspaces were used to enable Dagster tools to load user code in tools like Dagit or the CLI. In a `Definitions` world, workspaces are only required to load multiple code locations as part of an [open source (OSS) deployment](/deployment). For local development, `Definitions` users can utilize a [`pyproject.toml` file](/concepts/code-locations).
 
-The repositories in a workspace are loaded in a different process and communicate with Dagster system processes over an RPC mechanism. This architecture provides several advantages:
+<!-- Code locations in a workspace are loaded in a different process and communicate with Dagster system processes over an RPC mechanism. This architecture provides several advantages:
 
 - When there is an update to user code, Dagit can pick up the change without a restart.
-- You can use multiple repositories to organize jobs, but still work on all of your repositories using a single running Dagit.
-- The Dagit process can run in a separate Python environment from user code so job dependencies do not need to be installed into the Dagit environment.
-- Each repository can be sourced from a separate Python environment, so teams can manage their dependencies (or even their Python versions) separately.
+- You can use multiple code locations to organize jobs, but still work on all of your code locations using a single instance of Dagit.
+- The Dagit process can run in a separate Python environment from user code so job dependencies don't need to be installed into the Dagit environment.
+- Each code location can be sourced from a separate Python environment, so teams can manage their dependencies (or even their Python versions) separately. -->
```

Comment:
This section should be moved to the code locations section, actually. All these benefits apply to core concept.